roadmap.qmd

```diff
@@ -4,155 +4,213 @@ title: Roadmap
+#### Learning
+
+This would include courses, brief tutorials, and learning shorts. These
+would be developed as [open educational
+resources](https://en.wikipedia.org/wiki/Open_educational_resources).
+
+| Status | Learning resource |
+|-----------------------------------------------|------------------------|
+| {{< var status.wip >}} | [`rostools/git4cats`](https://github.com/rostools/git4cats): :cat: Git for Collaborative Teams: An introductory course on using Git and GitHub for team-based collaborative workflows and practices |
+| {{< var status.planned >}} | [`rostools/r-pkg-intro`](https://github.com/rostools/r-pkg-intro): Build research software using R as a solo developer in research environments. |
+| {{< var status.potential >}} | Course: Iterative project management and development for effective collaboration and work. |
+| {{< var status.potential >}} | Course: Introduction to building research software using Python as a solo developer in research environments. |
+| {{< var status.potential >}} | Course: Building research software in a team. |
+
+: "Wishlist" of learning-type documentation products we currently
+working on, will work on, or hope to work on.
+
+#### Goals
+
+This includes how-to guides, usage guides, and manuals.
+
+| Status | Guides or manuals |
+|-----------------------------------------------|------------------------|
+| {{< var status.wip >}} | [`sprout.seedcase-project.org`](https://sprout.seedcase-project.org/docs/guide) usage guide: Using and interacting with `seedcase-sprout`. |
+| {{< var status.planned >}} | Flower usage guide: Using and interacting with `seedcase-flower`. |
+| {{< var status.planned >}} | Propagate usage guide: Using and interacting with `seedcase-propagate`. |
+| {{< var status.planned >}} | Garden usage guide: Using and interacting with `seedcase-garden`. |
+| {{< var status.planned >}} | Sprout extensions usage guide: Installing or creating extensions for `seedcase-sprout`. |
+| {{< var status.planned >}} | Installing and setting up Seedcase software on server environments. |
+| {{< var status.planned >}} | Using Seedcase to create and manage data packages. |
+| {{< var status.planned >}} | A guided walkthrough: Team-based collaborative practices and workflows in a research environment. |
+| {{< var status.planned >}} | A guided walkthrough: Iterative and incremental project development and management in a research environment. |
+| {{< var status.planned >}} | Using DocOps workflows and infrastructure to reduce time between writing and dissemination. |
+| {{< var status.planned >}} | Building DataOps pipelines and infrastructure for improving research data quality and reduce time from collection to analysis. |
+| {{< var status.planned >}} | Improving the research software development lifecycle by integrating DevOps practices. |
+| {{< var status.potential >}} | Integrating research operations in organizations to improve quality and speed up research. |
```

Comment:
Most of these would come naturally as we develop the team and contributing docs. But maybe instead of the way we do it now, we could rearrange things so that the docs start working towards creating these outputs.

Reply:
That's because they are newly added to this project, but they've been bouncing around in my head for a long time :stuck_out_tongue: and yes, during the in-person sessions this week as well as visioning sessions later :grin: